drivedb: Add WD Re+ 5/6TB

esproul opened the pull request:
Small update for these SATA drives, there appear to only be two models. Attached is the smartctl report.

[smartctl-WDC-WD6005FRPZ.txt](https://github.com/user-attachments/files/22047206/smartctl-WDC-WD6005FRPZ.txt)

Here is a diff of smartctl output before and after this addition:
```
--- /dev/fd/63	2025-08-29 13:31:02.255398046 +0000
+++ /dev/fd/62	2025-08-29 13:31:02.255398046 +0000
@@ -2,6 +2,7 @@
 Copyright (C) 2002-20, Bruce Allen, Christian Franke, www.smartmontools.org

 === START OF INFORMATION SECTION ===
+Model Family:     Western Digital Re+
 Device Model:     WDC WD6005FRPZ-01S9PB0
 Serial Number:    WD-WX21D25R51UH
 LU WWN Device Id: 5 0014ee 05955f1db
@@ -10,7 +11,7 @@
 Sector Sizes:     512 bytes logical, 4096 bytes physical
 Rotation Rate:    5700 rpm
 Form Factor:      3.5 inches
-Device is:        Not in smartctl database [for details use: -P showall]
+Device is:        In smartctl database [for details use: -P show]
 ATA Version is:   ACS-3 T13/2161-D revision 3b
 SATA Version is:  SATA 3.1, 6.0 Gb/s (current: 6.0 Gb/s)
```

chrfranke commented:
- New entry should be moved below `Western Digital Re`.
- `tested ...` comment lacks firmware version, see the other entries.
- Please provide sample outputs for the other devices listed in the comment or remove these.
- `-v 16,...` option is missing. May be `Total_LBAs_Read` as in entry `Western Digital Re` , but value does not match `Device Statistics`.

```
Device Model:     WDC WD6005FRPZ-01S9PB0
...
ID# ATTRIBUTE_NAME          FLAGS    VALUE WORST THRESH FAIL RAW_VALUE
...
 16 Unknown_Attribute       -O---K   002   198   000    -    140826232365
...
Page  Offset Size        Value Flags Description
...
0x01  0x028  6    129699069451  ---  Logical Sectors Read
```

If possible, please check whether both values increase consistently on device reads.

----

PS: `Serial Number:    (redacted)`: Add `-q noserial` to suppress Serial and WWN.

esproul commented:
Addressed feedback, please see commit message.

> PS: `Serial Number: (redacted)`: Add `-q noserial` to suppress Serial and WWN.

Thank you for this tip. :)

chrfranke commented:
- Attribute 16 should be named `Unknown_WDC_Attribute` then.
- 2nd comment `// WDC WD6005FRPZ-01S9PB0` is redundant and should be removed.
